src/python/bot/fuzzers/syzkaller/engine.py

```diff
@@ -79,13 +79,10 @@ def prepare(self, corpus_dir, target_path, unused_build_dir):  # pylint: disable
     Returns:
       A FuzzOptions object."""
     self.prepare_binary_path()
-    config = runner.get_config()
+    args = runner.get_config()
+    args += ['--coverfile', runner.get_cover_file_path()]
```

Comment:
Since we are calling a helper in runner, why not just add this in get_config directly. We need this coveragefile in all cases.

Reply:
We only want it for fuzz, doesn't make sense to get it other places